config/environments/shared_deployment_config.rb

```diff
@@ -95,4 +95,5 @@
   # config.active_record.database_selector = { delay: 2.seconds }
   # config.active_record.database_resolver = ActiveRecord::Middleware::DatabaseSelector::Resolver
   # config.active_record.database_resolver_context = ActiveRecord::Middleware::DatabaseSelector::Resolver::Session
+  config.i18n.available_locales = [:en, :es]
```

Comment:
the other global i18n config settings are currently in application.rb (fallbacks, default_language). i don't have a strong opinion on where they belong, but it's probably nicer if they're all in one place.